GitCommands/RevisionReader.cs

```diff
@@ -97,13 +97,37 @@ public IReadOnlyCollection<GitRevision> GetRevisionsFromList(IList<ObjectId> unt
             return GetRevisionsFromArguments(arguments, cancellationToken);
         }
 
+        /// <summary>
+        /// Get the GitRevisions for all commits between 2 commits (boundaries included).
+        /// </summary>
+        /// <param name="olderCommitId">older commit Id.</param>
+        /// <param name="newerCommitId">newer commit Id.</param>
+        /// <param name="cancellationToken">Cancellation cancellationToken.</param>
+        /// <returns>List with GitRevisions.</returns>
+        public IReadOnlyCollection<GitRevision> GetRevisionsFromRange(string olderCommitId, string newerCommitId, CancellationToken cancellationToken)
+        {
+            if (string.IsNullOrWhiteSpace(olderCommitId) || string.IsNullOrWhiteSpace(newerCommitId))
+            {
+                return Array.Empty<GitRevision>();
+            }
+
+            GitArgumentBuilder arguments = new("log")
+                {
+                    "-z",
+                    $"--pretty=format:\"{FullFormat}\"",
+                    $"{olderCommitId}~..{newerCommitId}"
+                };
+
+            return GetRevisionsFromArguments(arguments, cancellationToken);
+        }
+
         /// <summary>
         /// Get the GitRevisions for Git argument.
         /// </summary>
         /// <param name="arguments">Git command arguments.</param>
         /// <param name="cancellationToken">Cancellation cancellationToken.</param>
         /// <returns>List with GitRevisions.</returns>
-        private IReadOnlyCollection<GitRevision> GetRevisionsFromArguments(GitArgumentBuilder arguments, CancellationToken cancellationToken)
+        public IReadOnlyCollection<GitRevision> GetRevisionsFromArguments(GitArgumentBuilder arguments, CancellationToken cancellationToken)
```

Comment:
why?

Reply:
Because i needed it for my PoC and I don't really understand how I forgot to revert it...
Will do it right away ;)